modules/service/src/main/scala/lucuma/odb/graphql/binding/ExecutionEventTypeBinding.scala

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.graphql
+package binding
+
+import lucuma.odb.data.ExecutionEventType
```

Comment:
I think I should probably move this to `core`?

Reply:
Probably, maybe as a follow-on since it's such a pain to change core.